docs/how-to/release_process/canary.rst

```diff
@@ -0,0 +1,107 @@
+Canary Testing for Checkbox Edge Version: In-depth Process
+==========================================================
+
+Introduction
+------------
+
+The following sections provide a detailed walkthrough of the Canary Testing process for the Checkbox Edge version, starting from snap building to the outcomes of the testing.
+
+.. toctree::
+   :maxdepth: 1
+
+   canary_pipeline
+   validation_job_example
+   validation_pipeline_execution
```

Comment:
I would put this at the very end of this page, since it's impossible to understand what these are without having a picture of the whole process first.

Or... just don't include this TOC, since the links here are also provided later on in the document, where they make more sense for the reader.

Reply:
In the current version it does not render on the page. I don't know if your comment still applies...

Reply:
I removed it completely and the page is exactly the same. Removed in d7f5fbe966c7994e02297e66a5584cb1619fc6e2